srtp/srtp.c

```diff
@@ -3473,24 +3785,37 @@ srtp_unprotect_rtcp(srtp_t ctx, void *srtcp_hdr, int *pkt_octet_len) {
       return srtp_err_status_no_ctx;
     } 
   }
-  
+
+  /*
+   * Determine if MKI is being used and what the MKI index is
```

Comment:
This returns the session keys given the MKI found in the packet.  It does not determine an MKI index, as the comment states.